tizen/src/backgroundWorker.h

```diff
@@ -0,0 +1,19 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_BACKGROUNDWORKER_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_BACKGROUNDWORKER_H_
+
+#include <flutter/method_call.h>
+#include <flutter/method_channel.h>
+#include <job_scheduler.h>
+
+#include <memory>
+
+#include "options.h"
+
+class BackgroundWorker {
```

Comment:
This could be used or not depend on specific usage of job scheduler and tizen service. I don't need it right now, but this could be needed in future. Can I remove this class when it is clear that I don't need this?

Reply:
I'd like a brief explanation in the comments.

Reply:
Removed the class